_includes/1.0/api/json/invoice.json

```diff
@@ -76,6 +76,7 @@
   "paid": 770.1,
   "unpaid": 370.1,
   "delivery_status": 1,
+  "invoice_status": 1,
```

Comment:
Test on your own data, but You should not be able to have an `invoice_status` of `1`. Only `2` or `3`